Program crash on start when deployed on Docker

**Description:**
When running a module with docker image built by simple @docker:Config with simple connection to rabbitmq through ballerina/rabbitmq my module crash and exit immediately: (from ballerina-internal.log):
`[2019-11-15 14:54:48,003] SEVERE {b7a.log.crash} - com/rabbitmq/client/ConnectionFactory
java.lang.NoClassDefFoundError: com/rabbitmq/client/ConnectionFactory
    at org.ballerinalang.messaging.rabbitmq.util.ConnectionUtils.createConnection(ConnectionUtils.java:61)
    at org.ballerinalang.messaging.rabbitmq.nativeimpl.connection.CreateConnection.createConnection(CreateConnection.java:48)
    at ballerina.rabbitmq.$value$Connection.createConnection(connection.bal:56)
    at ballerina.rabbitmq.$value$Connection.call(connection.bal)
    at ballerina.rabbitmq.$value$Connection.__init(connection.bal:24)
    at ballerina.rabbitmq.$value$Connection.call(connection.bal)
    at nowaera.product_service.___init.nowaera_product_service_0_1_0__init_(product_service:12)
    at nowaera.product_service.___init.$moduleInit(product_service:65535)
    at nowaera.product_service.___init.$lambda$$moduleInit$(product_service)
    at org.ballerinalang.jvm.scheduling.SchedulerItem.execute(Scheduler.java:426)
    at org.ballerinalang.jvm.scheduling.Scheduler.run(Scheduler.java:218)
    at org.ballerinalang.jvm.scheduling.Scheduler.runSafely(Scheduler.java:191)
    at org.ballerinalang.jvm.scheduling.Scheduler.start(Scheduler.java:178)
    at nowaera.product_service.___init.main(product_service)
Caused by: java.lang.ClassNotFoundException: com.rabbitmq.client.ConnectionFactory
    at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
    at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:349)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
    ... 14 more`
		
In opposite when running by "ballerina run" command, module creates connection correctly. I am sure that connection from the container to rabbitmq is working(ping test)
**Steps to reproduce:**
`import ballerina/rabbitmq;
rabbitmq:ConnectionConfiguration rabbit_config = {
 username: "username",
 password: "password",
 host:"localhost",
 port: 5672
};
rabbitmq:Connection rabbit_connection = new(rabbit_config);
rabbitmq:Channel publish_channel = new(rabbit_connection);`

**Affected Versions:**
1.0.3

**OS, DB, other environment details and versions:**
Docker on Windows

**Suggested Labels (optional):**
rabbitmq

--- a/stdlib/messaging/rabbitmq/src/main/ballerina/Ballerina.toml
+++ b/stdlib/messaging/rabbitmq/src/main/ballerina/Ballerina.toml
@@ -11,3 +11,10 @@ target = "java8"
     path = "./lib/ballerina-rabbitmq-@project.version@.jar"
     groupId = "ballerina"
     modules = ["rabbitmq"]
+
+    [[platform.libraries]]
+    artifactId = "amqp-client"
+    version = "5.7.3"
+    path = "./lib/amqp-client-5.7.3.jar"
+    groupId = "com.rabbitmq"
+    modules = ["rabbitmq"]

